internal/verify/retry.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/hashicorp/terraform-provider-aws/internal/tfresource"
 )
 
-func RetryOnAWSCode(code string, f func() (interface{}, error)) (interface{}, error) {
+func RetryOnErrCode(code string, f func() (interface{}, error)) (interface{}, error) {
```

Comment:
This function seems to a duplicate of `tfresource.RetryWhenAWSErrCodeEquals` with a hard-coded 2 minute timeout. Do you want to consolidate?